apps/backend/.env.sample

```
@@ -46,4 +46,7 @@ GOTENBERG_API_BASIC_AUTH_PASSWORD=...
 GOTENBERG_URL=...
 
 # Redis
-REDIS_URL=...
\ No newline at end of file
+REDIS_URL=...
+
+#Feature Flags
+FEATURE_FLAG_MCP_PARLA_ALLOWED=...
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)